### DISCUSSION TITLE
protobuf-c-1.4.1，required protobuf header file not found

### DISCUSSION THREAD

**Discussion by k7arm** [protobuf-c/protobuf-c#687] [Q&A]
Hello.I have already installed protobuf(23.4) in the default location.
/usr/local/include  and /usr/local/lib. But the protobuf-c(1.4.1) still can not found the protobuf header files. what should i do? Thanks.

checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether C compiler accepts "-Wc99-c11-compat"... yes
checking whether C compiler accepts "-Werror=incompatible-pointer-types"... yes
checking whether C compiler accepts "-Werror=int-conversion"... yes
checking whether C compiler accepts "-Wnull-dereference"... yes
checking for doxygen... no
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking whether g++ supports C++11 features with -std=c++11... yes
checking for protobuf... yes
checking google/protobuf/compiler/command_line_interface.h usability... no
checking google/protobuf/compiler/command_line_interface.h presence... no
checking for google/protobuf/compiler/command_line_interface.h... no
configure: error: required protobuf header file not found

**Comment by chenweihua** [comment]
如何解决

**Comment by k7arm** [comment]
https://gitlab.archlinux.org/archlinux/packaging/packages/protobuf-c/-/blob/main/PKGBUILD?ref_type=heads

https://patch-diff.githubusercontent.com/raw/protobuf-c/protobuf-c/pull/556.patch

patch -p1 < ../patch

**Reply by seven1240** [reply]
looks like the patch doesn't apply to master any more
